lib/processor/mongodb.go

```diff
@@ -28,10 +29,11 @@ func NewMongoDBConfig() MongoDBConfig {
 	rConf.Backoff.MaxElapsedTime = "30s"
 
 	return MongoDBConfig{
-		MongoDB:      client.NewConfig(),
-		Parts:        []int{},
-		Operation:    "insert",
```

Comment:
The default value here was incorrect (but maybe we shouldn't have any default set for the operation field?